document-service/doc-api/src/doc_api/__init__.py

```diff
@@ -0,0 +1,82 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Document API service.
+
+This module is the API for the BC Registries Document Service application.
+"""
+import os
+
+from flask import Flask
+from flask_cors import CORS
+from flask_migrate import Migrate, upgrade
+
+from doc_api import errorhandlers, models
+from doc_api.config import config
+from doc_api.metadata import APP_RUNNING_ENVIRONMENT
+from doc_api.models import db
+from doc_api.resources import meta_endpoint, ops_endpoint, v1_endpoint
+from doc_api.utils.auth import jwt
+from doc_api.utils.logging import logger, setup_logging
+
+setup_logging(os.path.join(os.path.abspath(os.path.dirname(__file__)), "logging.yaml"))  # important to do this first
+
+
+def create_app(service_environment=APP_RUNNING_ENVIRONMENT, **kwargs):
+    """Return a configured Flask App using the Factory method."""
+    app = Flask(__name__)
+    CORS(app)
+    app.config.from_object(config[service_environment])
+    app.url_map.strict_slashes = False
+
+    errorhandlers.init_app(app)
+
+    db.init_app(app)
+    Migrate(app, db)
```

Comment:
you sure you want to add migrate to init here?

Reply:
Removed.